VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/introduction/introduction.component.ts

```diff
@@ -3,7 +3,7 @@ import { PageUrls } from 'src/app/shared/page-url.constants';
 import { Router, ActivatedRoute } from '@angular/router';
 import { VideoWebService } from 'src/app/services/api/video-web.service';
 import {
-  UpdateParticipantStatusEventRequest, ConferenceResponse
+  UpdateParticipantStatusEventRequest, ConferenceResponse, EventType
```

Comment:
As mentioned before, this endpoint cannot be re-used for judges . if you look at the controller the reason is always set to joining.

Reply:
Reason amended as discussed and this can be resolved